modules/planning/tasks/deciders/BUILD

```diff
@@ -7,13 +7,15 @@ cc_library(
     srcs = [
         "decider_creep.cc",
         "decider_rule_based_stop.cc",
+        "path_assessment_decider.cc",
         "path_bounds_decider.cc",
         "side_pass_path_decider.cc",
         "side_pass_safety.cc",
     ],
     hdrs = [
         "decider_creep.h",
         "decider_rule_based_stop.h",
+        "path_assessment_decider.h",
```

Comment:
This lib gets too big. Please separate them.

Reply:
separated them.